app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -1911,6 +1911,309 @@ class DatabaseMigrationAndroidTest {
       assertThat(it.string(newColumnName)).isEqualTo("2017-09-25T11:20:42.008Z")
     }
   }
+
+  @Test
+  fun migrate_prescription_from_35_to_36() {
+    val db_v35 = helper.createDatabase(version = 35)
+
+    val tableName = "PrescribedDrug"
+    val uuid = "drug-uuid"
+    db_v35.execSQL("""
+      INSERT INTO $tableName VALUES(
+      '$uuid',
+      'drug',
+      'dosage',
+      'rxNormCode',
+      0,
+      1,
+      'patientUuid',
+      'facilityUuid',
+      'PENDING',
+      'created-at',
+      'updatedAt',
+      'null',
+      'recorded-at')
+    """)
+
+    val db_v36 = helper.migrateTo(version = 36)
+
+    db_v36.query("SELECT * FROM $tableName").use {
+      it.moveToNext()
+      assertThat(it.columnCount).isEqualTo(12)
+      assertThat(it.count).isEqualTo(1)
```

Comment:
I just wanted to keep all assertions together.
That makes it a little easier to see all the verification being performed.

Reply:
Done.

Reply:
Yeah, I understand what you tried to do here, but consider the fact that if you'd had to verify records on two or more rows instead of just one, it would've ended up looking something like:

```kotlin
it.moveToNext()
assertThat(it.count).isEqualTo(3)
assertThat(it.getColumnIndex("recordedAt")).isEqualTo(-1)
assertThat(it.string("uuid")).isEqualTo(uuid1)
it.moveToNext()
assertThat(it.string("uuid")).isEqualTo(uuid2)
it.moveToNext()
assertThat(it.string("uuid")).isEqualTo(uuid3)
```

Which kinds of breaks the vertical rhythm of reading the code since the first block is larger than the others. So if you break the assertions into two groups, one where you assert on the entire result set, and the other where you assert on the individual rows, the assertions read much nicer.

```kotlin
assertThat(it.count).isEqualTo(3)
assertThat(it.getColumnIndex("recordedAt")).isEqualTo(-1)

it.moveToNext()
assertThat(it.string("uuid")).isEqualTo(uuid1)
it.moveToNext()
assertThat(it.string("uuid")).isEqualTo(uuid2)
it.moveToNext()
assertThat(it.string("uuid")).isEqualTo(uuid3)
```

What do you think?